test/contract/unit/tickmap.test.ts

```diff
@@ -1,602 +1,621 @@
+//       expect(result).toBe(expected)
+//     }
+//     {
+//       const tick = GLOBAL_MAX_TICK - 3n
+//       const tickSpacing = 5n
+//       const up = true
+//       const result = (await invariant.view.getSearchLimit({ args: { tick, tickSpacing, up } }))
+//         .returns
+//       const expected = tick
+//       expect(result).toBe(expected)
+//     }
+//   })
+//   test('test next and prev intialized', async () => {
+//     // initialized edges
+//     for (let tickSpacing = 1n; tickSpacing <= 10n; tickSpacing++) {
+//       const feeTier = newFeeTier(0n as Percentage, tickSpacing)
+//       const poolKey = newPoolKey(token0Id, token1Id, feeTier)
+//       const invariant = await deployInvariant(sender, protocolFee)
+
+//       const maxIndex = GLOBAL_MAX_TICK - (GLOBAL_MAX_TICK % tickSpacing)
+//       const minIndex = -maxIndex
+
+//       {
+//         const [chunkIndex] = await tickToPosition(invariant, maxIndex, tickSpacing)
+
+//         await initializeChunk(invariant, sender, poolKey, chunkIndex)
+
+//         await flip(invariant, true, maxIndex, poolKey)
+//       }
+//       {
+//         const [chunkIndex] = await tickToPosition(invariant, minIndex, tickSpacing)
+
+//         await initializeChunk(invariant, sender, poolKey, chunkIndex)
+
+//         await flip(invariant, true, minIndex, poolKey)
+//       }
+//       const tickEdgeDiff = (TICK_SEARCH_RANGE / tickSpacing) * tickSpacing
+//       {
+//         const [isSome] = await nextInitialized(
+//           invariant,
+//           maxIndex - tickEdgeDiff,
+//           tickSpacing,
+//           poolKey
+//         )
+//         expect(isSome).toBeTruthy()
+//       }
+//       {
+//         const [isSome] = await prevInitialized(
+//           invariant,
+//           minIndex + tickEdgeDiff,
+//           tickSpacing,
+//           poolKey
+//         )
+//         expect(isSome).toBeTruthy()
+//       }
+//     }
+//     for (let tickSpacing = 1n; tickSpacing <= 10n; tickSpacing++) {
+//       const feeTier = newFeeTier(0n as Percentage, tickSpacing)
+//       const poolKey = newPoolKey(token0Id, token1Id, feeTier)
+//       const invariant = await deployInvariant(sender, protocolFee)
+
+//       const maxIndex = GLOBAL_MAX_TICK - (GLOBAL_MAX_TICK % tickSpacing)
+//       const minIndex = -maxIndex
+
+//       const tickEdgeDiff = (TICK_SEARCH_RANGE / tickSpacing) * tickSpacing
+//       {
+//         const [isSome] = await nextInitialized(
+//           invariant,
+//           maxIndex - tickEdgeDiff,
+//           tickSpacing,
+//           poolKey
+//         )
+//         expect(isSome).toBeFalsy()
+//       }
+//       {
+//         const [isSome] = await prevInitialized(
+//           invariant,
+//           minIndex + tickEdgeDiff,
+//           tickSpacing,
+//           poolKey
+//         )
+//         expect(isSome).toBeFalsy()
+//       }
+//     }
+//   }, 175000)
+// })
+
 describe('tickmap tests', () => {
```

Comment:
Have you checked if there is more elegant way to disable test from jest config?

Reply:
There is `skip`, but because the functions used here just do not exist commenting them out was the only option to not get a typescript compile time warning.